New feature

Leowolfking opened the pull request:
modify Document contents

Leowolfking committed:
add new documents

Leowolfking committed:
add picture

Leowolfking committed:
add new documents (#774)

* add new documents

* add picture

haoyuying committed:
hrnet_w18_samll_v1_humanseg and shufflenet_humanseg (#778)

ShenYuhan committed:
delete serving index.html (#802)

ShenYuhan committed:
print traceback for serving (#811)

ShenYuhan committed:
modify lac serving demo. (#824)

* modify lac serving demo.

* update lac_serving_demo.py

Leowolfking committed:
Merge branch 'develop' of https://github.com/PaddlePaddle/PaddleHub into new-feature

ShenYuhan committed:
fix gbk bug on windows (#836)

haoyuying committed:
Add sr (#808)

Leowolfking committed:
modify contents

Steffy-zxf commented on docs/install.md:
````diff
@@ -0,0 +1,100 @@
+> 如果出现`Your Paddle Fluid is installed successfully`，说明飞桨已成功安装。
+
+
+```python
````
这里应该写shell

Steffy-zxf commented on docs/install.md:
````diff
@@ -0,0 +1,100 @@
+Required-by:
+```
+
+## 如何卸载
````
为啥要教用户如何卸载hub呢[笑哭]

Steffy-zxf commented on docs/quick_experience/cmd_quick_run.md:
````diff
@@ -0,0 +1,184 @@
+
+
+```python
+# 指定版本安装PaddleHub，使用清华源更稳定、更迅速
````
这里就不要指定版本了，就用下面的最新版本安装吧

moguguo replied:
接纳，仅保留最新版本安装的命令。并同步修改python快速入门此部分。

Steffy-zxf commented on docs/quick_experience/cmd_quick_run.md:
````diff
@@ -0,0 +1,184 @@
+### 中文分词
+
+
+```python
````
shell

moguguo replied:
接纳，命令行的代码块注释统一修改为shell。修改了cmd_quick_run.md和python_use_hub.md

Steffy-zxf commented on docs/quick_experience/cmd_quick_run.md:
```diff
@@ -0,0 +1,184 @@
+
+### PaddleHub命令行工具简介
+
+PaddleHub的命令行工具在开发时借鉴了Anaconda和PIP等软件包管理的理念，可以方便快捷的完成模型的搜索、下载、安装、升级、预测等功能。 下面概要介绍一下PaddleHub支持的12个命令，详细介绍可查看[命令行参考](http://)章节。：
```
命令行参考是一个空链接？

moguguo replied:
接纳，修改为准确链接

T-baby committed:
Delete some duplicate codes (#832)

- Delete some duplicate codes
- Fix the problem of not being able to process unlogged words

moguguo committed:
Update more_demos.md (#850)

Leowolfking committed:
update contents

Leowolfking committed:
Merge branch 'develop' of https://github.com/PaddlePaddle/PaddleHub into new-feature

nepeplwu committed:
Update how_to_finetune.md

Leowolfking committed:
Merge branch 'release/v1.8' of https://github.com/PaddlePaddle/PaddleHub into new-feature

Leowolfking committed:
Merge branch 'release/v1.8' of https://github.com/PaddlePaddle/PaddleHub into new-feature

Leowolfking committed:
Merge branch 'new-feature' of https://github.com/Leowolfking/PaddleHub into new-feature
